src/content/docs/logs/ui-data/parsing.mdx

```diff
@@ -330,6 +339,53 @@ Watch our YouTube video (approx. 6 minutes) to learn how to use a Grok debugger
     - `json({"dropOriginal": false})`: This will show the JSON payload that was extracted. When set to `false`, the full JSON-only payload will be displayed under an attribute named in `my_attribute_prefix` above. Note the JSON attributes will remain in the message field here as well giving the user 3 different views of the JSON data. If storage of all three versions is a concern it's recommended to use the default of `true` here.
     - `json({"depth": 62})`: Levels of depth you want to parse the JSON value (defaulted to 62).
   </Collapser>
+  <Collapser
+    id="parsing-csv"
+    title="Parsing CSV"
+  >
+    If your system is sending CSV logs and you need to parse them in New Relic, you can use `csv` [grok type](#grok-syntax), which will parse the CSV captured by the grok pattern.
+    This format relies on 3 main parts: the grok syntax, the prefix you would like to assign to the parsed json attributes, and the `csv` [grok type](#grok-syntax). Using the `csv` [grok type](#grok-syntax), you can extract and parse CSV from logs.
+
+    Given the following CSV log line as an example:
+
+    ```
+    "2015-05-13T23:39:43.945958Z,202,POST,/shopcart/checkout,142,10"
+    ```
+
+    And a parsing rule with the following shape:
+
+    ```
+    %{GREEDYDATA:log:csv({"columns": ["timestamp", "status", "method", "url", "time", "bytes"]})
```

Comment:
hm, just occurred to me -- what is the `log` name used for?

Reply:
Wop... nice point, all the keys will be prefixed with `log` unless the `noPrefix: true` configuration was set.